next-env.d.ts

```diff
@@ -1,10 +1,6 @@
 /// <reference types="next" />
 /// <reference types="next/types/global" />
+/// <reference types="next/image-types/global" />
 
-import { Env } from './types';
-
-declare global {
-    export namespace NodeJS {
-        export interface ProcessEnv extends Env {}
-    }
-}
```

Comment:
I don't see this part being moved anywhere else.
I'd add a type declaration at `/@types/index.d.ts`

Reply:
Added as [`@types/global.d.ts`](https://github.com/prezly/theme-nextjs-starter/pull/21/commits/1ae4b7a4c05a66fe03d0b0261dae32d6198c51d6) 👍